framework/include/araui/ui_animation.h

```diff
@@ -42,14 +50,102 @@ typedef enum {
 extern "C" {
 #endif
 
+/**
+ * @brief Create a move animation
+ *
+ * If you want to get a current coordinate of widget, use ui_widget_get_rect.
+ *
+ * @param[in] from_x The x coordinate to move from
+ * @param[in] from_y The y coordinate to move from
+ * @param[in] to_x The x coordinate to move to
+ * @param[in] to_y The y coordinate to move to
+ * @param[in] duration Duration of animation
+ * @param[in] intrp_type Interpolation of animation
```

Comment:
I think `Interpolation type of the animation` is better

Reply:
Fixed